clients/privacy-center/app/server-environment.ts

```diff
@@ -355,6 +357,9 @@ export const loadPrivacyCenterEnvironment =
         .FIDES_PRIVACY_CENTER__ALLOW_HTML_DESCRIPTION
         ? process.env.FIDES_PRIVACY_CENTER__ALLOW_HTML_DESCRIPTION === "true"
         : null,
+      BASE_64_COOKIE: process.env.BASE_64_COOKIE
+        ? process.env.BASE_64_COOKIE === "true"
```

Comment:
I think this should also have the `FIDES_PRIVACY_CENTER__` prefix, right?

Reply:
ah good catch!